.github/CONTRIBUTING.md

```diff
@@ -9,6 +9,37 @@ Unit tests can be run via `npm run test:unit` command.
 
 To run the code, a GitHub PR against `develop` should be raised with the committed code to the branch PR. The PR runs deployment script with deploy to development environment. The script builds the code that's added as part of your change and installs it in Azure DevOps organization as an extension that can be added to run a pipeline.
 
+### Local debugging
+
+A number of environment variable are required for debugging, here's an example launch config for `VSCode` that sets mandatory parameters such as `AGENT_TEMPDIRECTORY`, `INPUT_failOnIssues` and `INPUT_authToken`
+
+```
+{
```

Comment:
I've tried to run this configuration and it didn't work for me. I was getting exit code 9 from `snykTestToolRunner.exec()`. Did it work for you without any additional setup?

I've noticed you are using Windows, whereas I am on Mac. If that works on Windows, would be good to note it here.

Reply:
Exit code 9 from that call is the error I mentioned regarding `internalModuleReadJSON is not a function` . 

Full root cause is https://blog.sethcorker.com/question/getting-to-the-bottom-of-mix-tailwind-exited-with-1/, summarized as '[Packaged apps crash with zero messaging when NODE_OPTIONS is set](https://github.com/vercel/pkg/issues/1517)' and VSCode setting NODE_OPTIONS on launch. 

Setting to `null` seems to do the trick, will include that in next change